src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.InProcess.cs

```diff
@@ -182,6 +182,10 @@ internal abstract partial class AbstractNavigateToSearchService
             foreach (var match in nameMatches)
                 matchedSpans.AddRange(match.MatchedSpans);
 
+            using var allPatternMatches = TemporaryArray<PatternMatch>.Empty;
+            allPatternMatches.AddRange(containerMatches);
+            allPatternMatches.AddRange(nameMatches);
```

Comment:
new aiosp allows us to pass all the individual matches we made (for example, if user searched for `Con.Wri` and we matched `Console` and `WriteLine`).   So passing this data along here.